### PR TITLE
Some suggested new activities

### PR DESCRIPTION
**Pull request by Perlkonig**
For your consideration.

### CODE REVIEW

File: activities.json [hunk 1]
```diff
@@ -166,3 +166,31 @@
 {"activity":"Learn about a distributed version control system such as Git","accessibility":0.0,"type":"education","participants":1,"price":0.0,"link":"","key":"9303608","enabled":true}
 {"activity":"Write a note of appreciation to someone","accessibility":0.0,"type":"social","participants":1,"price":0.0,"link":"","key":"1770521","enabled":true}
 {"activity":"Draw something interesting","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"","key":"8033599","enabled":true}
+{"activity":"Learn and play a new card game","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"https://www.pagat.com","key":"9660022","enabled":true}
+{"activity":"Write a list of things you are grateful for","accessibility":0.0,"type":"relaxing","participants":1,"price":0.0,"link":"","key":"2062010","enabled":true}
+{"activity":"Organize one of your collections","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"6098037","enabled":true}
```
Comment (drewthoennes) [inline activities.json:171]: What did you mean by collections?

File: activities.json [hunk 2]
```diff
@@ -166,3 +166,31 @@
...
+{"activity":"Color","accessibility":0.0,"type":"relaxation","participants":1,"price":0.1,"link":"","key":"5322987","enabled":true}
+{"activity":"Organize your pantry","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"3954882","enabled":true}
+{"activity":"Back up important computer files","accessibility":0.2,"type":"busywork","participants":1,"price":0.2,"link":"","key":"9081214","enabled":true}
+{"activity":"Memorize a favourite quote or poem","accessibility":0.8,"type":"education","participants":1,"price":0.0,"link":"","key":"9008639","enabled":true}
+{"activity":"Do something you used to do as a kid","accessibility":0.8,"type":"relaxation","participants":1,"price":0.1,"link":"","key":"8827573","enabled":true}
+{"activity":"Organize your basement","accessibility":0.9,"type":"busywork","participants":1,"price":0.0,"link":"","key":"8203595","enabled":true}
+{"activity":"Write a todo list for the coming weekend","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"9487940","enabled":true}
```
Comment (drewthoennes) [inline activities.json:184]: There's already "Make a to-do list for your week". This seems too similar and I'm not sure if we can change it enough to make it justifiably different.

File: activities.json [hunk 3]
```diff
@@ -166,3 +166,31 @@
...
 {"activity":"Draw something interesting","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"","key":"8033599","enabled":true}
+{"activity":"Learn and play a new card game","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"https://www.pagat.com","key":"9660022","enabled":true}
+{"activity":"Write a list of things you are grateful for","accessibility":0.0,"type":"relaxing","participants":1,"price":0.0,"link":"","key":"2062010","enabled":true}
+{"activity":"Organize one of your collections","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"6098037","enabled":true}
+{"activity":"Learn Morse code","accessibility":0.0,"type":"education","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Morse_code","key":"3646173","enabled":true}
+{"activity":"Go for a walk","accessibility":0.1,"type":"relaxation","participants":1,"price":0.0,"link":"","key":"4286250","enabled":true}
+{"activity":"Mow your neighbour's lawn","accessibility":0.2,"type":"charity","participants":1,"price":0.0,"link":"","key":"1303906","enabled":true}
```
Comment (drewthoennes) [inline activities.json:174]: For consistency, can we use the Americanized spelling of neighbor?

File: activities.json [hunk 4]
```diff
@@ -166,3 +166,31 @@
...
+{"activity":"Organize your pantry","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"3954882","enabled":true}
+{"activity":"Back up important computer files","accessibility":0.2,"type":"busywork","participants":1,"price":0.2,"link":"","key":"9081214","enabled":true}
+{"activity":"Memorize a favourite quote or poem","accessibility":0.8,"type":"education","participants":1,"price":0.0,"link":"","key":"9008639","enabled":true}
+{"activity":"Do something you used to do as a kid","accessibility":0.8,"type":"relaxation","participants":1,"price":0.1,"link":"","key":"8827573","enabled":true}
+{"activity":"Organize your basement","accessibility":0.9,"type":"busywork","participants":1,"price":0.0,"link":"","key":"8203595","enabled":true}
+{"activity":"Write a todo list for the coming weekend","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"9487940","enabled":true}
+{"activity":"Sit in the dark and listen to your favourite music with no distractions","accessibility":1.0,"type":"relaxation","participants":1,"price":0.0,"link":"","key":"9908721","enabled":true}
```
Comment (drewthoennes) [inline activities.json:185]: Can we use favorite for consistency?

File: activities.json [hunk 5]
```diff
@@ -166,3 +166,31 @@
...
+{"activity":"Match your storage containers with their lids","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"4940907","enabled":true}
+{"activity":"Start a family tree","accessibility":1.0,"type":"social","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Family_tree","key":"6825484","enabled":true}
+{"activity":"Look at your finances and find one way to save money","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"5977626","enabled":true}
+{"activity":"Organize your dresser","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"7556665","enabled":true}
+{"activity":"Learn the periodic table","accessibility":0.6,"type":"education","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Periodic_table","key":"3621244","enabled":true}
+{"activity":"Learn how or improve your touch typing","accessibility":0.8,"type":"busywork","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Touch_typing","key":"2526437","enabled":true}
```
Comment (drewthoennes) [inline activities.json:194]: Did you mean to say "Learn how to improve your touch typing"?

**Comment by drewthoennes** [comment]
I just added a few comments for your review. Once we work through those, this should be good to go!

File: activities.json (hunk 1 above)
Comment (Perlkonig) [reply activities.json:171]: I mean like a collection like books, games, stamps, etc. Are you OK with an "(e.g., ...)" clause?

File: activities.json (hunk 2 above)
Comment (Perlkonig) [reply activities.json:184]: I somehow missed that, Sorry. Will remove that from the commit.

File: activities.json (hunk 3 above)
Comment (Perlkonig) [reply activities.json:174]: Sorry. Habit. Will fix in next commit.

File: activities.json (hunk 4 above)
Comment (Perlkonig) [reply activities.json:185]: Yup!

File: activities.json (hunk 5 above)
Comment (Perlkonig) [reply activities.json:194]: I'll just go with "improve." I was trying to say "Learn if you don't already know or improve if you already do."

File: activities.json (hunk 1 above)
Comment (drewthoennes) [reply activities.json]: Ah I understand. Personally, I've avoided that for simplicity's sake and I'd prefer to keep it consistent. Maybe we can split this into "Organize the books on your bookshelf", "Organize a cluttered drawer", etc?

Comment (Perlkonig) [reply activities.json]: Works for me! Will push new commit later.

**Comment by drewthoennes** [comment]
Looks great to me! Thanks for making all of those changes.

**Pull request merged**